templates/shopify-bigquery/assets/shopify.asset.yml

```diff
@@ -0,0 +1,8 @@
+name: raw.shopify
```

Comment:
It's an orders table, so please name it orders. Also, we need other source_tables as well. So for each source_table, we need an asset

Reply:
And instead of raw, maybe we can call it shopify_raw.orders

Reply:
done